### PR TITLE
Created NRT_PrivlegedRoleAssignedOutsidePIM.yaml

### PR DESCRIPTION
**Pull request by samikroy**
Change(s):
   - Created NRT_PrivlegedRoleAssignedOutsidePIM.yaml

   Reason for Change(s):
   - New NRT rule addition.

   Version Updated:
   - Initial Version

   Testing Completed:
   - Yes

### CODE REVIEW

**Comment by samikroy** [comment]
@aprakash13  - Please let me know for any updates needed to this rule.

**Comment by vmanojreddy** [comment]
@aprakash13  Please review and provide your feedback. thanks!!!

File: Detections/AuditLogs/NRT_PrivlegedRoleAssignedOutsidePIM.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+description: |
+  'Identifies a privileged role being assigned to a user outside of PIM
+  Ref : https://docs.microsoft.com/azure/active-directory/fundamentals/security-operations-privileged-accounts#things-to-monitor-1'
+severity: Low
+requiredDataConnectors:
+  - connectorId: AzureActiveDirectory
+    dataTypes:
+      - AuditLogs
+tactics:
+  - PrivilegeEscalation
```
Comment (aprakash13) [inline Detections/AuditLogs/NRT_PrivlegedRoleAssignedOutsidePIM.yaml:12]: Please add the relevantTechniques

**Review by aprakash13 — CHANGES_REQUESTED** [review]
Small change suggested. Please modify.

**Review by aprakash13 — APPROVED** [review]
Thanks @samikroy for all your contributions.

**Comment by samikroy** [comment]
Thank you for the approval & merge @aprakash13 .

**Pull request merged**